sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/storage/AuthTokenStorage.java

```diff
@@ -33,12 +38,16 @@
     String getHomeAccountId();
 
     /**
-     * Removes token value.
+     * Gets the oldest token info from history.
+     *
+     * @return auth token info.
      */
-    void removeToken();
+    AuthTokenInfo getOldestToken();
```

Comment:
Why do we need a getOldestToken?

Reply:
To send the logs with the oldest token first. During sending we're interesting only for the oldest one, no need to have a full history outside of this class.

I'll add more description in Javadoc.